src/algorithms/time_evolution/evoltools.jl

```diff
@@ -1,28 +1,28 @@
 """
-    get_gate(dt::Float64, H::LocalOperator)
+    get_expham(dt::Float64, H::LocalOperator)
 
-Compute `exp(-dt * H)` from the nearest neighbor Hamiltonian `H`.
+Compute `exp(-dt * H)` from Hamiltonian `H`.
+Each term in `H` must be a single `TensorMap`.
```

Comment:
This is a bit misleading, given that this is not actually the full exponential of the hamiltonian, but rather a weird intermediate struct we use that represents `sum(exp(-h * dt) for h in H)` but is not used that way. (in particular, `exp(sum)` would be `prod(exp)`, not `sum(exp)`). I don't have any issues with reusing `LocalOperator` that way, I would just reword the docstring.